### DISCUSSION TITLE
will not record .

### DISCUSSION THREAD

**Discussion by peter247** [blakeblackshear/frigate#2990] [General]
Trying to run frigate on a lower powered computer , but can get it to record .
I've tried use my old set up config.yml but can get it to record anything .
It's made the clips and record directory , which the clips are being saved to , but record directory is blank.

I've found why looking at the logs :- 
2022-03-19 17:04:43] ffmpeg.driveway.detect         ERROR   : Error writing trailer of /tmp/cache/driveway-%Y%m%d%H%M%S.mp4: No space left on device
The video-file.mp4 gets bigger and bigger until it runs out of memory , so it is not making small segments .

mqtt:
  
  host: -------
  port: 1883
  topic_prefix: frigate
  client_id: frigate
  user: mqtt_user
  password: -------
  stats_interval: 60

cameras:
  driveway:
    ffmpeg:
      hwaccel_args:
        - -hwaccel
        -  vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
        #- -hwaccel_output_format
        #- yuv420p

      input_args: ["-avoid_negative_ts","make_zero","-fflags","+genpts+discardcorrupt","-rtsp_transport","tcp","-stimeout","5000000","-use_wallclock_as_timestamps","1","-flags","low_delay"]

      inputs:
        - path: rtsp://admin:rusty247@192.168.0.14:554/h264Preview_01_main
          roles:
            - record
            - rtmp
            - detect

    detect:                 
      #width: 640
      #height: 480
      fps: 4
    objects:
      track:
        - person
       # - car
        - dog
        - bicycle
        - cat
        - bird
        - horse
      filters:
        person:
          threshold: 0.7
        dog:
          threshold: 0.6
        cat:
          threshold: 0.6

    snapshots:
      enabled: true
      timestamp: false
      bounding_box: true
      retain:
        default: 5

    record:
      enabled: true
      retain_days: 0  
      events:
        retain:
          default: 10  
 
detectors:
  cpu1:
    type: cpu
    num_threads: 3
  cpu2:
    type: cpu
    num_threads: 3
  cpu3:
    type: cpu
    num_threads: 3
  cpu4:
    type: cpu
    num_threads: 3

**Comment by NickM-27** [comment]
Have you triggered any events? You have retain days set to 0 so it isn't going to but anything in the recording directory unless there's an event. 

Just saw the logs, you may need to increase the cache size

**Reply by peter247** [reply]
I made a VM / docker install , used the record lines from someone else's config.yml and it worked first time .
So I'm going to delete the install on docker / bare metal of proxmox use that config file and see if it works.
Testing if I can run 3 cameras on a intel j4125 , 2 x h264 and 1 x h265 camera and that running at around 75% cpu , but I what to get the hardware acceleration.
THE CACHE SIZE , if I set it to 32gb it will just slowly keep increasing until it runs out , I set the cache to 4 times to normal and still get the same problem .

**Comment by peter247** [comment]
I've found how to get it to working , but not sure why ?
I mounted a second drive to /media/frigate and it worked right away with the same config.yml .
